src/platform_impl/linux/event_loop.rs

```diff
@@ -39,10 +39,14 @@ use crate::{
 use super::{
   keyboard,
   monitor::{self, MonitorHandle},
-  util,
+  taskbar, util,
   window::{WindowId, WindowRequest},
 };
 
+use taskbar::TaskbarIndicator;
+
+static mut TASKBAR: Option<TaskbarIndicator> = None;
```

Comment:
I think this could be stored on `EventLoop` type instead of a static variable

Reply:
which event listener?

Reply:
This part 
https://github.com/ahqsoftwares/tao/blob/e232c00378659c3c9a0e7b88b9b5cd4baf47f31f/src/platform_impl/linux/event_loop.rs#L315-L378

Reply:
you can store it in `Rc<RefCell>>` or `Rc<Mutex>` and pass it to the handler

Reply:
I shall try this today night

Reply:
WSL Green Signal
Just left testing the outputs